.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "git.alwaysSignOff": true
+    "git.alwaysSignOff": true,
```

Comment:
@Elijah699 will you kindly remove this file? We can skip committing it in this PR.

Reply:
Yes, let me do that rightaway